charts/smokey/templates/cronjob.yaml

```diff
@@ -3,7 +3,12 @@
 apiVersion: batch/v1
 kind: CronJob
 metadata:
-  name: {{ .name }}
+  name: {{- include "smokey.name" . -}}
```

Comment:
Does this produce valid output with the `-}}` (trim whitespace from right-hand side) there? Haven't run it myself but I suspect it may elide the newline and produce something like `name: smokeylabels:` (not sure tho, just caught my eye as a bit sus)